test/common/utxobased/engine/engine.spec.ts

```diff
@@ -286,30 +286,21 @@ describe('engine.spec', function () {
     describe(`Get Transactions from Wallet type ${WALLET_TYPE}`, function () {
       it('Should get number of transactions from cache', function () {
         assert.equal(
-          engine.getNumTransactions({}),
+          engine.getNumTransactions({ tokenId: null }),
           TX_AMOUNT,
           `should have ${TX_AMOUNT} tx from cache`
         )
       })
 
       it('Should get transactions from cache', async function () {
-        await engine.getTransactions({}).then(txs => {
+        await engine.getTransactions({ tokenId: null }).then(txs => {
           assert.equal(
             txs.length,
             TX_AMOUNT,
             `should have ${TX_AMOUNT} tx from cache`
           )
         })
       })
-
-      it('Should get transactions from cache with options', async function () {
-        const options: EdgeGetTransactionsOptions = {
-          startIndex: 1,
-          startEntries: 2
-        }
-        const txs = await engine.getTransactions(options)
-        assert.equal(txs.length, 2, 'should have 2 tx from cache')
-      })
```

Comment:
Why was this test removed?

Reply:
See above. The core removed the ability to pass any `getTransactions` flags besides the `tokenId`. Since we don't need these features, we don't need to test them.